tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
WinObjC.Frameworks.targets are the consumption rules for the WinObjc.Frameworks package, so yes you want to think about how to generalize what is going on there to work for middleware.

---
In reply to: [108994382](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108994382) [](ancestors = 108994382,108993288)